### PR TITLE
aligning controls in the bar chart dialogue

### LINKED ISSUE #5815: Small changes in Describe > Specific > Bar Chart dialogue
There is an odd label on this dialogue, namely **_Position:_**.  This should move to be above the pull-down that currently says Dodge.  
Second is to align the controls on the right or the dialogue.  Currently the control of the position is not aligned with the others

### PR DESCRIPTION
**Pull request by Ivanluv**
@africanmathsinitiative/developers  this is ready for review..Fixes #5815

### CODE REVIEW

**Comment by lloyddewit** [comment]
@rdstern OK from my side, please can you test? If your tests pass then @maxwellfundi can merge.

**Comment by Ivanluv** [comment]
@rdstern could you kindly look at this again

File: instat/dlgBarAndPieChart.vb [hunk 1]
```diff
@@ -350,4 +352,12 @@ Public Class dlgBarAndPieChart
...
         TestOkEnabled()
     End Sub
+
+    Private Sub CoreControls_ContentsChanged(ucrChangedControl As ucrCore) Handles ucrSaveBar.ControlContentsChanged, ucrReceiverY.ControlContentsChanged, ucrReceiverFirst.ControlContentsChanged
+
+    End Sub
+
+    Private Sub ucrPnlOptions_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlOptions.ControlValueChanged
+
+    End Sub
```
Comment (Patowhiz) [inline instat/dlgBarAndPieChart.vb:362]: @Ivanluv I can see we have 2 extra empty subs. Did you add them intentionally, if not then I think it's good to have them deleted

**Comment by lloyddewit** [comment]
Approved from my side!
@maxwellfundi Please could you merge? thanks

**Comment by shadrackkibet** [comment]
@rdstern can you approve this?

**Comment by lloyddewit** [comment]
@rdstern if you approve again then we can merge, thanks

**Pull request merged**